mir-ci/mir_ci/test_drag_and_drop.py

```diff
@@ -30,7 +35,15 @@
         apps.mir_demo_server(),
     ],
 )
-@pytest.mark.deps(debs=("libgtk-4-dev",), pip_pkgs=(("pygobject", "gi"), ("robotframework~=6.1.1", "robot")))
+@pytest.mark.deps(
+    debs=("libgtk-4-dev",),
```

Comment:
To be adapted for maybe a different font, but:
```suggestion
    debs=("libgtk-4-dev", "fonts-cantarell", "adwaita-icon-theme"),
```